modules/core/shared/src/main/scala/gem/EphemerisKey.scala

```diff
@@ -6,6 +6,7 @@ package gem
 import gem.enum.EphemerisKeyType
 import gem.parser.EphemerisKeyParsers
 import gem.syntax.parser._
+import gem.syntax.string._
```

Comment:
So we use `mouse` only for strings?

Reply:
Yes, and it doesn't provide everything we need (`parseBigDecimal` for instance). I would consider contributing all this back but for now it's so minor it seems like less of a hassle to just maintain it ourselves. We can contribute general stuff back once we have a better idea of what we're going to need.